src/typing.py

```diff
@@ -247,7 +247,7 @@ def __subclasscheck__(self, cls):
         raise TypeError("Forward references cannot be used with issubclass().")
 
     def __repr__(self):
-        return '_ForwardRef(%r)' % (self.__forward_arg__,)
+        return repr(self.__forward_arg__)
```

Comment:
@gvanrossum OK, I reverted this part in a new commit, I don't have a strong preference here.